django_cloud_deploy/workflow/__init__.py

```diff
@@ -266,6 +266,7 @@ def create_and_deploy_new_project(
             self._enable_service_workflow.enable_required_services(
                 project_id, required_services)
 
+        static_content_dir = settings.STATIC_ROOT
```

Comment:
I don't understand how this is equivalent to static_content_dir = os.path.join(django_directory_path, 'static') 

Couldn't django directory path be in another location than the current directory? I thought python manage.py collectstatic is for the current directory.